### DISCUSSION TITLE
RustDesk Server Install Script - gohttpserver Change Authentcation pass

### DISCUSSION THREAD

**Discussion by ghost** [rustdesk/rustdesk#2640] [Q&A]
In regards to the script https://github.com/techahold/rustdeskinstall
Which nicely installs a self-hosted version. 
IT creates a gohttp web server and posts WindowsAgentAIOInstall.ps1 Powershell script to install the clients later.
I am trying to figure out how to change the authentication password set on the site it creates?
I see the rustdeskinstall/install.sh script references ${admintoken} as the password but not sure how that is generated or changed. 

Here is the area of the install script that sets it up.

# Setup Systemd to launch Go HTTP Server
gohttpserver="$(cat << EOF
[Unit]
Description=Go HTTP Server
[Service]
Type=simple
LimitNOFILE=1000000
ExecStart=/opt/gohttp/gohttpserver -r ./public --port 8000 --auth-type http --auth-http admin:**${admintoken}**
WorkingDirectory=/opt/gohttp/
User=${uname}
Group=${uname}
Restart=always
StandardOutput=append:/var/log/gohttp/gohttpserver.log
StandardError=append:/var/log/gohttp/gohttpserver.error
# Restart service after 10 seconds if node service crashes
RestartSec=10
[Install]
WantedBy=multi-user.target
EOF
)"
echo "${gohttpserver}" | sudo tee /etc/systemd/system/gohttpserver.service > /dev/null
sudo systemctl daemon-reload
sudo systemctl enable gohttpserver.service
sudo systemctl start gohttpserver.service

**Comment by chrfischer** [comment]
The password should be in the file /etc/systemd/system/gohttpserver.service in the line starting with "ExecStart".
Just edit and restart the service (or the server).
